indexer/src/indexer.ts

```diff
@@ -488,6 +489,176 @@ export const createIndexerService = (config: IndexerDbCredentials) => {
+    const dayInSecondsSql = sql.raw("86400");
+
+    const query = sql`
+        WITH LatestBalances AS (SELECT p.pool_address,
+                                       p.token0_denom,
+                                       p.token0_balance,
+                                       p.token1_denom,
+                                       p.token1_balance,
+                                       p.height,
+                                       ROW_NUMBER() OVER (PARTITION BY p.pool_address ORDER BY p.height DESC) as rn
+                                FROM v1_cosmos.materialized_pool_balance p
+                                WHERE p.pool_address = ${poolAddressesSql}
+                                    ${createHeightsFilterSql("p", startHeight, endHeight)}),
+             PoolBalances AS (SELECT lb.pool_address,
+                                     lb.token0_denom,
+                                     lb.token0_balance,
+                                     lb.token1_denom,
+                                     lb.token1_balance,
+                                     lb.height
+                              FROM LatestBalances lb
+                              WHERE lb.rn = 1),
+             SwapVolumes AS (SELECT s.pool_address,
+                                    SUM(
+                                            CASE
+                                                WHEN s.offer_asset = pb.token0_denom THEN s.offer_amount
+                                                ELSE 0
+                                                END
+                                    ) AS token0_volume,
+                                    SUM(
+                                            CASE
+                                                WHEN s.offer_asset = pb.token1_denom THEN s.offer_amount
+                                                ELSE 0
+                                                END
+                                    ) AS token1_volume
+                             FROM v1_cosmos.materialized_swap s
+                                      JOIN PoolBalances pb ON s.pool_address = pb.pool_address
+                             WHERE s.pool_address = ${poolAddressesSql}
+                                 ${createHeightsFilterSql("s", startHeight, endHeight)}
+                             GROUP BY s.pool_address),
+             DailyYield AS (SELECT hpy.pool_address,
+                                   CASE
+                                       WHEN hpy.total_liquidity_usd > 0 AND
+                                            (EXTRACT(EPOCH FROM (LEAD(hpy.timestamp, 1, NOW())
+                                                                 OVER (PARTITION BY hpy.pool_address ORDER BY hpy.timestamp) -
+                                                                 hpy.timestamp)) / 86400) > 0
+                                           THEN hpy.fees_usd / hpy.total_liquidity_usd / (EXTRACT(EPOCH FROM (
```

Comment:
A nice add would be to also add the incentive APR on the pool right now, that does need changes on the view I believe

Reply:
Ok, will look into it

Reply:
Later as discussed